editor.planx.uk/src/pages/FlowEditor/components/Team/tests/helpers/userEntersInput.tsx

```diff
@@ -0,0 +1,13 @@
+import { fireEvent, within } from "@testing-library/react";
+
+export const userEntersInput = async (
+  labelText: string,
+  inputString: string,
+  container: HTMLElement,
+) => {
+  const inputField = await within(container).findByLabelText(labelText);
+
+  fireEvent.change(inputField, {
+    target: { value: inputString },
+  });
```

Comment:
Originally I had `user.type(inputField, inputString)` here (based on feedback from @DafyddLlyr in the last PR I opened) but I just couldn't get it working with this. 🤷 Probably something to dig into again soon - maybe in the upcoming testing discussion.